plugin.py

```diff
@@ -29,9 +30,26 @@ def is_allowed_to_start(
     ) -> Optional[str]:
         if not workspace_folders:
             return "Requires a folder to start."
-        path = workspace_folders[0].path
-        tailwind_config_file_path = os.path.join(path, 'tailwind.config.js')
-        if not os.path.exists(tailwind_config_file_path):
-            return "No tailwind.config.js present in {}".format(path)
-        return None
 
+        # Config pattern is found here:
+        # https://github.com/tailwindlabs/tailwindcss-intellisense/blob/766a5d533dcb68640ce6b3270488f6701dd1173d/packages/vscode-tailwindcss/src/extension.ts#L40
+        config_file_pattern = '(tailwind|tailwind.config).(js|cjs)'
+        folder_exclude_patterns = sublime.load_settings('Preferences.sublime-settings').get("folder_exclude_patterns", []) # type: List[str]
+        folder_exclude_patterns.append('node_modules') # definitely exclude node_modules
+        config_file = find_file_in_workspace(config_file_pattern, workspace_folders[0].path, folder_exclude_patterns)
+        if config_file:
+            return None # config found, return None to start the session
+        return "No tailwind configuration file present in the workspace folder."
+
+
+def find_file_in_workspace(file_pattern: str, root_folder: str, folder_exclude_patterns: Optional[List[str]] = None) -> Optional[str]:
+    for path, directories, files in os.walk(root_folder):
+        if folder_exclude_patterns:
+            for folder_excldue_pattern in folder_exclude_patterns:
+                # skip ignored folders
+                directories[:] = [d for d in directories if not re.search(folder_excldue_pattern, d)]
```

Comment:
This is a bit odd. the folder_exclude_pattern is a glob, but we're using it as if it's a regex.